docs/rules/quotes.md

```diff
@@ -1,12 +1,13 @@
 # Enforce Quote Style (quotes)
 
-Enforces coding style that string literals are delimited with single or double quotes.
+Enforces coding style that string literals are delimited with backticks, single or double quotes.
 
-It takes an option as the second parameter which can be `"double"` or `"single"` for double-quotes or single-quotes respectively. There is no default.
+It takes an option as the second parameter which can be `"double"`, `"single"` or `"back"` for double-quotes, single-quotes or backticks respectively. There is no default.
```

Comment:
We should really make the option "backtick". The extra four characters really help to make the option name self-explanatory.